CONTRIBUTING.md

```diff
@@ -11,6 +11,161 @@ requirements and recommendations.
+```
+
+### Open Folder
+
+- Navigate to the cloned folder:
+
+```shell
+cd opentelemetry-demo-webstore/
+```
+
+### Gradle Update [Windows Only]
+
+- Navigate to the Java Ad Service folder to install and update Gradle:
+
+```shell
+cd .\src\adservice\
+.\gradlew installDist
+.\gradlew wrapper --gradle-version 7.4.2
+```
+
+### Run Docker Compose
+
+- Start the demo (It can take ~20min the first time the command is executed as
+all the images will be build):
+
+```shell
+docker compose up
+```
+
+### Verify the Webstore & the Telemetry
+
+Once the images are built and containers are started you can access:
+
+- Webstore: <http://localhost:8080/>
+
+- Jaeger: <http://localhost:16686/>
+
+- Prometheus: <http://localhost:9090/>
+
+- Grafana: <http://localhost:3000/>
+
+## Create Your First Pull Request
+
+### How to Send Pull Requests
+
+Everyone is welcome to contribute code to `opentelemetry-dotnet` via GitHub pull
+requests (PRs).
+
+To create a new PR, fork the project in GitHub and clone the upstream repo:
+
+```sh
+git clone https://github.com/open-telemetry/opentelemetry-demo-webstore.git
+```
+
+Navigate to the repo root:
+
+```sh
+cd opentelemetry-dotnet
+```
+
+Add your fork as an origin:
+
+```sh
+git remote add fork https://github.com/open-telemetry/opentelemetry-demo-webstore.git
+```
+
+Check out a new branch, make modifications and push the branch to your fork:
+
+```sh
+$ git checkout -b feature
+# edit files
+$ git commit
+$ git push fork feature
+```
+
+Open a pull request against the main `opentelemetry-demo-webstore` repo.
+
+### How to Receive Comments
+
+- If the PR is not ready for review, please mark it as
+  [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
+- Make sure CLA is signed and all required CI checks are clear.
+- Submit small, focused PRs addressing a single
+  concern/issue.
+- Make sure the PR title reflects the contribution.
+- Write a summary that helps understand the change.
+- Include usage examples in the summary, where applicable.
+- Include benchmarks (before/after) in the summary, for contributions that are
+  performance enhancements.
+
+### How to Get PRs Merged
+
+A PR is considered to be **ready to merge** when:
+
+- It has received approval from
+  [Approvers](https://github.com/open-telemetry/community/blob/main/community-membership.md#approver).
+  /
+  [Maintainers](https://github.com/open-telemetry/community/blob/main/community-membership.md#maintainer).
+- Major feedbacks are resolved.
+- It has been open for review for at least one working day. This gives people
+  reasonable time to review.
+- Trivial change (typo, cosmetic, doc, etc.) doesn't have to wait for one day.
+- Urgent fix can take exception as long as it has been actively communicated.
+
+Any Maintainer can merge the PR once it is **ready to merge**. Note, that some
+PRs may not be merged immediately if the repo is in the process of a release and
```

Comment:
I guess this might not apply to the demo project?

Reply:
I'm assuming you mean the urgent fix section. Just removed it

Reply:
Or also the repo release comment?